tests/tests_manifests/data/min.yml

```diff
@@ -1 +1 @@
-schema-version: '1.0'
+schema-version: ''
```

Comment:
This should be reverted, a minimal manifest is one that has a schema version.

I think you need 3 tests:

- schema-version: no value -> raises an exception
- schema-version: empty string -> raises an exception
- schema-version: something -> works